edx_sysadmin/constants.py

```diff
@@ -0,0 +1,43 @@
+"""
+Constants for edx_sysadmin
+"""
+
+from django import forms
+
+from edx_sysadmin.utils.utility import (
+    get_country_choices,
+    get_gender_choices,
+    get_level_of_education_choices,
+    get_valid_year_of_birth_choices,
+)
+
+
+FIELDS_AND_DEFAULT_VALUES_MAP = {
+    # Pattern of Fields Mapping
```

Comment:
@HamzaIbnFarooq could you please remove commented code?

Reply:
Removed.